src/botprovisioner.ts

```diff
@@ -32,6 +32,7 @@ export class BotProvisioner {
 		}
 		const roomId = event.room_id;
 		const sender = event.sender;
+		const maxMsgSize = 4000;
```

Comment:
can you maybe make this a global constant on top in all caps? Like, outside the class `const MAX_MSG_SIZE = 4000;`